src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.ShowAdvanced": "Afficher Avancé",
+  "FSC.HideAdvanced": "Masquer Avancé",
+  "FSC.DateControls": "Contrôles de date",
+  "FSC.DateControlsHelp": "Utilisez ces commandes pour modifier la date actuelle dans votre jeu.",
+  "FSC.NoDateControls": "Simple Calendar a été configuré pour laisser un autre module contrôler la date et l'heure. Veuillez utiliser ce module pour mettre à jour l'heure ou modifier la configuration de Simple Calendars.",
+  "FSC.SetCurrentDate": "Définir la date actuelle",
+  "FSC.SetCurrentDateDialog.Title": "Confirmer le changement de date actuelle",
+  "FSC.SetCurrentDateDialog.Content": "Vous avez choisi de changer la date en un jour qui n'est actuellement pas visible !<br/>Voulez-vous vraiment changer la date du jour en <strong>{DATE}</strong> ?",
+  "FSC.Apply": "Appliquer",
+  "FSC.AdvancedOptions": "Options Avancées",
+  "FSC.Today": "Ajourd'hui",
+  "FSC.Configure": "Configurer",
+  "FSC.Import": "Importer",
+  "FSC.Export": "Exporter",
+  "FSC.By": "Par",
+  "FSC.Cancel": "Annuler",
+  "FSC.Edit": "Éditer",
+  "FSC.Delete": "Supprimer",
+  "FSC.DeleteConfirm": "Confirmer Suppression",
+  "FSC.DeleteConfirmText": "Vous voulez vraiment supprimer cette note?",
+  "FSC.OverwriteConfirm": "Écraser Existant",
+  "FSC.OverwriteConfirmText": "L'application de cette modification écrasera toutes les configurations de calendrier existantes. Êtes-vous sûr?",
+  "FSC.ChangeNextMonth": "Passer au mois suivant",
+  "FSC.ChangePreviousMonth": "Passer au mois précédent",
+  "FSC.MoveYearForward": "Avancer d'un an",
+  "FSC.MoveMonthForward": "Avancer d'un mois",
+  "FSC.MoveDayForward": "Avancer d'un jour",
+  "FSC.MoveYearBack": "Recul d'un an",
+  "FSC.MoveMonthBack": "Recul d'un mois",
+  "FSC.MoveDayBack": "Retour d'un jour",
+  "FSC.Time.Controls": "Contrôles de temps",
+  "FSC.Time.ControlsHelp": "Sélectionnez une unité de temps à modifier puis utilisez les commandes pour faire avancer ou réduire cette unité de 1 ou 5",
+  "FSC.Time.Forward": "Avancer le temps.",
+  "FSC.Time.Backward": "Déplacer le temps en arrière.",
+  "FSC.Time.Start": "Commencer à avancer le temps de jeu",
+  "FSC.Time.Stop": "Arrêter d'avancer le temps de jeu",
+  "FSC.Notes.DialogTitle": "Note",
+  "FSC.Notes.New": "Nouvelle note",
+  "FSC.Notes.Empty": "Aucune note pour cette date",
+  "FSC.Notes.For": "Notes pour",
+  "FSC.Notes.AddNew": "Ajouter une nouvelle note",
+  "FSC.Notes.NewFor": "Nouvelle note pour",
+  "FSC.Notes.Title": "Titre de la note",
+  "FSC.Notes.TitleHelp": "Le titre de cette note.",
+  "FSC.Notes.Date": "Date de la note",
+  "FSC.Notes.DateHelp": "La date de la note.",
+  "FSC.Notes.NotPlayerVisible": "Non visible au Joueur",
+  "FSC.Notes.PlayerVisible": "Visible au Joueur",
+  "FSC.Notes.PlayerVisibleHelp": "Si cette note est visible par les joueurs ou non.",
+  "FSC.Notes.Reminder": "Rappelez-moi!",
+  "FSC.Notes.ReminderHelp": "Cet événement vous sera rappelé.",
+  "FSC.Notes.Repeating": "Répétitions de notes",
```

Comment:
I am getting inspired by Google Calendar for this one: I think the correct way to say this one would be "Récurence de la note"